### PR TITLE
Implementation of the Counting Sort algorithm in Python.

### PR DESCRIPTION
**Pull request by sahebsunny**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

counting sort is an algorithm for sorting a collection of objects according to keys that are small integers.
Worst complexity: n+r
Average complexity: n+r
Space complexity: n+r

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

I want to keep contributing to NeoAlgo.
Thank you.

### CODE REVIEW

File: Python/sort/Counting_Sort.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        output[count[arr[i]] - 1] = arr[i]
+        count[arr[i]] -= 1
+        i -= 1
+
+    # Copy the sorted elements into original array
+    for i in range(0, size):
+        arr[i] = output[i]
+
+
+lst = [6, 4, 2, 5, 3, 4, 1]
```
Comment (atarax665) [inline Python/sort/Counting_Sort.py:31]: Dynamic input is preferred.

File: Python/sort/Counting_Sort.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Implementation of the Counting Sort algorithm in Python.
```
Comment (atarax665) [inline Python/sort/Counting_Sort.py:1]: Add a short description of Counting sort with its space and time complexity.

File: Python/sort/Counting_Sort.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    # Copy the sorted elements into original array
+    for i in range(0, size):
+        arr[i] = output[i]
+
+
+lst = [6, 4, 2, 5, 3, 4, 1]
+counting_Sort(lst)
+print("Sorted arr in Ascending Order: ")
+print(lst)
```
Comment (atarax665) [inline Python/sort/Counting_Sort.py:34]: Add sample Input and Output in commented form.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/sort/Counting_Sort.py (hunk 3 above)
Comment (sahebsunny) [reply Python/sort/Counting_Sort.py:34]: Thank you sir for your review
I am making a new Pr

Comment (sahebsunny) [reply Python/sort/Counting_Sort.py:34]: Pr no #1878
@atarax665

**Comment by atarax665** [comment]
#1878

**Pull request closed without merging**